scripts/eod_ecosystem_update.py

```
@@ -19,31 +19,40 @@
     """End of Day ecosystem update orchestrator."""
-    
+
     def __init__(self):
         """Initialize the EOD updater."""
-        self.ecosystem_service = get_ecosystem_ingestion_service()
-        self.memos_client = get_memos_client()
+        try:
+            self.ecosystem_service = get_ecosystem_ingestion_service()
+            self.memos_client = get_memos_client()
+        except Exception as e:
+            logger.error(f"Failed to initialize EOD updater: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)